server/routes/openConditions.js

```diff
@@ -26,29 +26,28 @@ module.exports = function Index({ formService, offendersService, userService, au
     asyncMiddleware(async (req, res, transactionalDbClient) => {
       const { bookingId } = req.params
       const form = 'furtherCharges'
-      const result = await buildFormData(res, req, 'openConditions', form, bookingId, transactionalDbClient)
+      let result = await buildFormData(res, req, 'openConditions', form, bookingId, transactionalDbClient)
```

Comment:
Generally in js it is preferred to have immutable objects, probs that's why this code was using the newResult variable further down the line. Making it mutable simplifies the code though, so I think the change make sense.